addon/marks/inline-rdfa.ts

```diff
@@ -16,14 +17,16 @@ export const inline_rdfa: MarkSpec = {
       getAttrs(node: HTMLElement) {
         const attrs = getRdfaAttrs(node);
         if (attrs) {
-          return attrs;
+          return { ...attrs, _guid: uuidv4() };
         }
         return false;
       },
     },
   ],
   toDOM(mark: Mark) {
-    return ['span', mark.attrs, 0];
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const { _guid, ...rdfaAttrs } = mark.attrs;
```

Comment:
I'll add some tests, I don't think `spanning: false` is needed...

Reply:
If I would enable `spanning: false`, it would break all nested rdfa: e.g.:
```html
<span resource="a">
foo 
<span property="ext:bar">baz</span>
qux
</span>
```
would be serialized to:
```html
<span resource="a">
foo 
</span>
<span resource="a">
<span property="ext:bar">baz</span>
</span>
<span resource="a">
qux
</span>
```
which is precisely what we want to prevent, so `spanning: true` is absolutely necessary.

What is interesting is that prosemirror will only merge the marks in the serialized output when they have the same attributes, so:
```html
<p>
   <span resource='a'>ab</span>
   <span resource='a'>cd</span>
</p>
```
still serializes in:
```html
<p>
   <span resource='a'>ab</span>
   <span resource='a'>cd</span>
</p>
```
As the `inline_rdfa` marks have different guid's.

Reply:
oh ok I misunderstood our previous conversation then